core/session.ts

```diff
@@ -103,20 +103,6 @@ export class Session {
         "Actions may only contain more than one SQL statement if they are of type 'operations'."
       );
     }
-    if (
```

Comment:
Why remove all these?

Reply:
your excess-property checks catch these - so it's just a double-error.